libkbfs/kbfs_ops_test.go

```diff
@@ -3372,347 +3367,3 @@ func TestForceFastForwardOnEmptyTLF(t *testing.T) {
-		}
-		if len(children) != 1 {
-			t.Fatalf("Unexpected children: %v", children)
-		}
-		if _, ok := children[newName]; !ok {
-			t.Fatalf("Child %s missing: %v", newName, children)
-		}
-	}
-	checkChild(config, rootNode)
-
-	// Sync everything.
-	err = kbfsOps.SyncAll(ctx, rootNode.GetFolderBranch())
-	if err != nil {
-		t.Fatalf("Couldn't sync; %+v", err)
-	}
-
-	// Check that bob can see it.
-	config2 := ConfigAsUser(config, "bob")
-	defer CheckConfigAndShutdown(ctx, t, config2)
-
-	rootNodeBob := GetRootNodeOrBust(ctx, t, config2, "alice,bob", false)
-	checkChild(config2, rootNodeBob)
-
-	if len(ops.blocks.deCache) > 0 {
-		t.Fatalf("%d unexpected deCache entries leftover",
-			len(ops.blocks.deCache))
-	}
-}
-
-func TestKBFSOpsSetExSyncAll(t *testing.T) {
-	config, _, ctx, cancel := kbfsOpsInitNoMocks(t, "alice", "bob")
-	defer kbfsTestShutdownNoMocks(t, config, ctx, cancel)
-
-	rootNode := GetRootNodeOrBust(ctx, t, config, "alice,bob", false)
-	kbfsOps := config.KBFSOps()
-	name := "myfile"
-	fileNode, _, err := kbfsOps.CreateFile(ctx, rootNode, name, false, NoExcl)
-	if err != nil {
-		t.Fatalf("Couldn't create file: %+v", err)
-	}
-
-	// Manually set the executability of a file without actually
-	// syncing it.  TODO(KBFS-2076) remove all this duplicated code.
-	ops := getOps(config, rootNode.GetFolderBranch().Tlf)
-	{
-		rootDir := ops.nodeCache.PathFromNode(rootNode)
-		filePath := ops.nodeCache.PathFromNode(fileNode)
-
-		sao, err := newSetAttrOp(name, rootDir.tailPointer(),
-			exAttr, filePath.tailPointer())
-		if err != nil {
-			t.Fatal(err)
-		}
-
-		lState := makeFBOLockState()
-		dblock, err := ops.blocks.GetDirBlockForReading(
-			ctx, lState, ops.head, rootDir.tailPointer(), ops.branch(), rootDir)
-		if err != nil {
-			t.Fatal(err)
-		}
-		de := dblock.Children[name]
-		de.Type = Exec
-
-		_ = ops.blocks.SetAttrInDirEntryInCache(lState, filePath, de, sao.Attr)
-		ops.dirOps = append(ops.dirOps, cachedDirOp{sao, []Node{fileNode}})
-	}
-
-	// Make sure we can't see it before the sync happens.
-	checkChild := func(config Config, rootNode Node) {
-		children, err := config.KBFSOps().GetDirChildren(ctx, rootNode)
-		if err != nil {
-			t.Fatal(err)
-		}
-		if len(children) != 1 {
-			t.Fatalf("Unexpected children: %v", children)
-		}
-		if ei, ok := children[name]; !ok {
-			t.Fatalf("Child %s missing: %v", name, children)
-		} else if ei.Type != Exec {
-			t.Fatalf("Unexpected type: %s", ei.Type)
-		}
-	}
-	checkChild(config, rootNode)
-
-	// Sync everything.
-	err = kbfsOps.SyncAll(ctx, rootNode.GetFolderBranch())
-	if err != nil {
-		t.Fatalf("Couldn't sync; %+v", err)
-	}
-
-	// Check that bob can see it.
-	config2 := ConfigAsUser(config, "bob")
-	defer CheckConfigAndShutdown(ctx, t, config2)
-
-	rootNodeBob := GetRootNodeOrBust(ctx, t, config2, "alice,bob", false)
-	checkChild(config2, rootNodeBob)
-
-	if len(ops.blocks.deCache) > 0 {
-		t.Fatalf("%d unexpected deCache entries leftover",
-			len(ops.blocks.deCache))
-	}
-}
```

Comment:
Yes, that's why they're being removed.  Coverage went up with this PR:

Master:
```
libkbfs$ go test -coverPASS
coverage: 66.4% of statements
ok  	github.com/keybase/kbfs/libkbfs	7.832s
```

This PR:
```
libkbfs$ go test -cover
PASS
coverage: 66.7% of statements
ok  	github.com/keybase/kbfs/libkbfs	7.848s
```